src/mpyl/utilities/repo/__init__.py

```diff
@@ -146,17 +152,16 @@ def changes_in_branch(self) -> list[Revision]:
         if not revisions:
             return []
 
-        first_revision = (
-            revisions[0].hexsha if len(revisions) != 1 else self._config.main_branch
-        )
```

Comment:
We still need this part I think

Reply:
Yeah I wasn't sure, but in which way though?

Reply:
The same I think.. if there's only one commit, we should diff with main_branch and that one revision

Reply:
It works fine with only one commit, I tested it with a second branch